### PR TITLE
feat: add version print flag for pulsarctl

### PR DESCRIPTION
**Pull request by ericsyh**
### Why for change
The pulsarctl loses the version print function.

### What did i change
I added the `--version | -V` flag to the pulsarctl. Using like below: 

```
./bin/pulsarctl

Usage: pulsarctl [command] [flags]

Commands:
  bookkeeper            Operations about bookKeeper
  broker-stats          Operations to collect broker statistics
  brokers               Operations about broker(s)
  clusters              Operations about cluster(s)
  completion            Generates shell completion scripts
  context               Interface for setting Pulsar Context
  functions             Interface for managing Pulsar Functions (lightweight, Lambda-style compute processes that work with Pulsar)
  functions-worker      Operations to collect function-worker statistics
  help                  Help about any command
  namespaces            Operations about namespaces
  ns-isolation-policy   Operations about namespace isolation policy
  oauth2                Operations about oauth2
  plugin                Operations about plugins
  resource-quotas       Operations about resource quotas
  schemas               Operations related to Schemas associated with Pulsar topics
  sinks                 Interface for managing Pulsar IO sinks (egress data from Pulsar)
  sources               Interface for managing Pulsar IO Sources (ingress data into Pulsar)
  subscriptions         Operations about subscription(s)
  tenants               Operations about tenant(s)
  token                 Operations of token
  topics                Operations about topic(s)

Common flags:
  -s, --admin-service-url string           The admin web service url that pulsarctl connects to.
      --audience string                    OAuth 2.0 audience identifier.
      --auth-params string                 Authentication parameters are used to configure the authentication provider specified by "AuthPlugin".
                                            Tls example: "tlsCertFile:val1,tlsKeyFile:val2"
                                            Token example: "authParams=file:///path/to/token/file" or "authParams=token:tokenVal"
      --auth-plugin string                 AuthPlugin is used to specify the plugin to use for authentication,
                                            the supported values are "org.apache.pulsar.client.impl.auth.AuthenticationTls"
                                            and "org.apache.pulsar.client.impl.auth.AuthenticationToken"
      --bookie-service-url string          The bookie web service url that pulsarctl connects to. (default "http://localhost:8080")
      --client-id string                   OAuth 2.0 client identifier.
  -C, --fabulous string                    toggle colorized logs (true,false,fabulous) (default "true")
  -h, --help                               help for this command
      --issuer-endpoint string             OAuth 2.0 issuer endpoint.
      --key-file string                    Path to the private key file.
      --tls-allow-insecure                 Allow TLS insecure connection
      --tls-cert-file string               File path for TLS cert used for authentication
      --tls-enable-hostname-verification   Enable TLS hostname verification
      --tls-key-file string                File path for TLS key used for authentication
      --tls-trust-cert-path string         Allow TLS trust cert file path
      --token string                       Using the token to authentication
      --token-file string                  Using the token file to authentication
  -v, --verbose int                        set log level, use 0 to silence, 4 for debugging (default 3)
  -V, --version                            Print version information and exit.

Use 'pulsarctl [command] --help' for more information about a command.
```

```
./bin/pulsarctl -V

Release Version: v2.8.0-rc-202105140121-22-g975857c
Git Commit Hash: 975857c25ef26721af4313cb6ffbef5f16a4dbcf
Git Branch: flag-version
UTC Build Time: 2021-06-26 06:28:42
Go Version: go version go1.16.3 darwin/amd64
```

```
./bin/pulsarctl --version

Release Version: v2.8.0-rc-202105140121-22-g975857c
Git Commit Hash: 975857c25ef26721af4313cb6ffbef5f16a4dbcf
Git Branch: flag-version
UTC Build Time: 2021-06-26 06:28:42
Go Version: go version go1.16.3 darwin/amd64

```

### CODE REVIEW

**Review by zymap — COMMENTED** [review]
Thanks for your contribution! Looks that the `bin` directory is unused. Could you please remove it?

**Comment by ericsyh** [comment]
> Thanks for your contribution! Looks that the `bin` directory is unused. Could you please remove it?

Sure, i will update the `Makefile` to build the pulsarctl to the root directory.

**Comment by ericsyh** [comment]
@zymap Updated in https://github.com/streamnative/pulsarctl/pull/383/commits/be30f591f35063493c075f3a427388b552d8c2c6. PTAL

**Comment by zymap** [comment]
I mean in your commits file there has a file `bin/pulsarct` which is unused. Could you please remove it?

**Comment by ericsyh** [comment]
@zymap I see. Just remove the `bin/pulsarctl` and add the rule in `.gitignore` .

File: pkg/pulsarctl.go [hunk 1]
```diff
@@ -56,12 +56,16 @@ func NewPulsarctlCmd() *cobra.Command {
...
+				cmdutils.PrintVersionInfo()
+				return
+			}
 			if err := cmd.Help(); err != nil {
 				logger.Debug("ignoring error %q", err.Error())
 			}
 		},
 	}
-
+	rootCmd.PersistentFlags().BoolP("version", "V", false, "Print version information and exit.")
```
Comment (zymap) [inline pkg/pulsarctl.go:68]:
```suggestion
	rootCmd.PersistentFlags().BoolP("version", "V", false, "show the pulsarctl version informantion")
```

**Review by zymap — APPROVED** [review]

**Pull request merged**